events_test.go

```diff
@@ -260,6 +264,36 @@ func TestAddRemoveHandlersMidStreamSync(t *testing.T) {
 	}
 }
 
+func TestPanicOnListener(t *testing.T) {
+	type Panic struct {
+		Message string
+	}
+
+	d := NewDispatcher[TestEvent]()
+
+	d.AddEventHandler(func(event TestEvent) {
+		t.Logf("Handler One: [Event: %s]\n", event.Message)
```

Comment:
Is the intend of the test case to panic for all three event handler? 

Asking the question for my own learning.

Reply:
Yes, this tests panic within the event handler and the dispatcher recovery. Because you can pass `panic` anything, we need a test to verify that our recovery code is resilient. For reference, here's the recovery code for the panic:

https://github.com/kubecost/events/blob/5d085fc7be3c7c10fb4baae781be835a898519df/dispatcher.go#L606-L621

Reply:
nice. Makes sense! TYVM!